frontend/src/metabase-lib/metadata.ts

```diff
@@ -1,11 +1,7 @@
 import * as ML from "cljs/metabase.lib.js";
 import * as ML_MetadataCalculation from "cljs/metabase.lib.metadata.calculation";
-import type {
-  DatabaseId,
-  DatasetColumn,
-  Field,
-  TableId,
-} from "metabase-types/api";
+import type { DatabaseId, DatasetColumn, TableId } from "metabase-types/api";
+import type Field from "./metadata/Field";
```

Comment:
Wrong `Field` type was being referenced here, I didn't realize it earlier because `Lib.fromLegacyColumn` was only used in JS files.

Reply:
other places use `import type Field from "metabase-lib/metadata/Field";`. I'd stick to such format of imports, but if you have any other opinion, please share

Reply:
I tend to use relative imports where it's possible to do so without "going up" (`../`).
Here it's consistent with `Metadata` import in line below.